exercises/E-conditionals/exercise.js

```diff
@@ -6,8 +6,14 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-let name = "Daniel";
-let danielsRole = "mentor";
+if (danielsRole === "mentor")
+{
+    console.log("Hi, I'm "+ name + ", I'm a " + danielsRole + ".")
+}
+else if (danielsRole === "student")
+[
+    console.log("Hi, I'm "+ name + ", I'm a " + danielsRole + ".")
+]
```

Comment:
Is there any point to use the if statement if we are going to use the same return statement ?